client/components/Map.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from "react";
+import MapView from 'react-native-maps';
+import { 
+    StyleSheet, 
+} from 'react-native';
+import MyMarker from "./MyMarker";
+import BikeMarker from "./BikeMarker";
+import { bikes } from '../data/testBikes';
+import { getBikes } from "../services/bikes";
+
+type LocationProps = {
+    latitude: number,
+    longitude: number
+}
+
+const Map = (props: LocationProps) => {
+
+    const [bikeArray, setBikeArray] = useState([]);
+
+    useEffect(() => {
+        getBikes()
+            .then(data => setBikeArray(data.bikes))
+            .catch(err => console.log(err));
+    }, [])
+
+    return (
+        <MapView 
+            style={styles.map}
+            region={{
+                latitude: props.latitude,
+                longitude: props.longitude,
+                latitudeDelta: 0.0922,
```

Comment:
What do _latitudeDelta_ and _longitudeDelta_ represent and why these values? Or are these just temporarily hardcoded?

Reply:
According to [apple.developer.com](https://developer.apple.com/documentation/mapkit/mkcoordinatespan/1452573-longitudedelta) those properties represent the amount of east-to-west and north-to-south distance to display on the map. The values in there now were part of some example code I found so we can definitely fiddle with them.